src/components/GradientInput/index.tsx

```diff
@@ -57,7 +57,7 @@ const GradientInput = (props: GradientInputProps): JSX.Element => {
                 path: selectedOption.strength_path,
                 default: (1 - currVal),
                 min: selectedOption.strength_min || 0,
-                max: selectedOption.strength_max || 1,
+                max: selectedOption.strength_max || 0.99,
```

Comment:
when strength_max is 1, we end up with a divide by 0 error when packing, so we're making it 0.99 instead!